src/modeler/AutoRest.Swagger/Validation/ParameterNameValidation.cs

```diff
@@ -32,7 +32,7 @@ public class ParameterNameValidation : TypedRule<SwaggerParameter>
         /// <summary>
         /// The severity of this message (ie, debug/info/warning/error/fatal, etc)
         /// </summary>
-        public override Category Severity => Category.Error;
```

Comment:
this doesn't look to be a naming convention rule, I think we need to skip this one.